src/Microsoft.Fx.Portability.Offline/Data.cs

```diff
@@ -63,6 +64,20 @@ public static IEnumerable<BreakingChange> LoadBreakingChanges()
             }
         }
 
+        private static IEnumerable<string> GetLocalAllowedCategories()
+        {
+            // Check to see if a file defining valid categories exists
+            var categoriesPath = Path.Combine(GetCurrentDirectoryName(), "BreakingChanges", "BreakingChangeCategories.json");
+            if (File.Exists(categoriesPath))
+            {
+                using (var categoriesFile = File.Open(categoriesPath, FileMode.Open, FileAccess.Read))
+                {
+                    return categoriesFile.Deserialize<string[]>();
+                }
+            }
+            return null;
```

Comment:
Instead of returning null, would it be better to return Enumerable.Empty<string>()? Then no null checking is needed